### PR TITLE
Ezmode

### PR DESCRIPTION
**Pull request by dmarx**
```python
# ta dah!
from mmc.ez.CLIP import clip

# mhm.
clip.available_models()

# requesting a tokenizer before loading the model
# returns the openai clip SimpleTokenizer
#tokenize = clip.tokenize

# either of these works
model, preprocessor = clip.load('RN50')
model, preprocessor = clip.load('[clip - openai - RN50]')

# if we request the tokenizer *after* a model has been loaded, 
# the tokenizer appropriate to the loaded model is returned 
tokenize = clip.tokenize
```

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -0,0 +1,16 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+torch = "*"
+Pillow = "*"
+mmc = {editable = true, path = "."}
+clip-anytorch = "*"
```
Comment (rom1504) [inline Pipfile:10]: I'd recommend specifying a major version here

File: Pipfile.lock [hunk 2]
```diff
@@ -0,0 +1,717 @@
+{
+    "_meta": {
+        "hash": {
```
Comment (rom1504) [inline Pipfile.lock:3]: I think you should remove the lock file
Lock files do not make sense for libraries: you want your CI to be as close as possible to the user experience, IE pip file.

File: Pipfile (hunk 1 above)
Comment (dmarx) [reply Pipfile:10]: could you recommend a minimal major version for pillow and/or torch? I keep getting into trouble setting versions for those packages in mmc specifically. I especially want to make sure installing mmc doesn't trigger uninstall of a working gpu-torch and replacing it with a cpu only version.

**Comment by dmarx** [comment]
oy vey, I didn't realize I left this unmerged. I'll make sure to do a solid chunk of mmc dev work this coming week, don't want this project to stagnate.

**Comment by dmarx** [comment]
clipfa still broken, but I think that was an issue before i even opened this commit. lgtfm

**Pull request merged**